### PR TITLE
News View and Media list view mode for issue/130

### PR DESCRIPTION
**Pull request by smayden**
Creating initial view for news matching the Media List Page (/news) in QS1

## Description
This is ready to review. In the workshop we:
- Styled the media list view mode on the news content type with updated bootstrap classes
- Changed the image style for the photo to card on the media list view mode
- Replaced short title with the full title in the media list view mode
- Renamed the display on the view to "Paged Row"

## Related Issue
#130 

## How Has This Been Tested?
Tested locally

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I've added this Pull Request to the AZ Quickstart project in the right column.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by danahertzberg** [comment]
@smayden We could probably use the card crop for images for now. It could be a good idea to look at other classes that achieve a similar look and already exist (`card` perhaps?). And the token issue is a widespread D9 issue, right?

File: modules/custom/az_core/config/install/core.entity_view_mode.node.az_media_list.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.az_media_list
+label: 'Media List'
+targetEntityType: node
+cache: true
```
Comment (trackleft) [inline modules/custom/az_core/config/install/core.entity_view_mode.node.az_media_list.yml:9]:
```suggestion
cache: true

```

File: modules/custom/az_news/az_news.info.yml [hunk 2]
```diff
@@ -24,4 +24,5 @@ dependencies:
   - az_paragraphs_cards
   - az_paragraphs_text
   - smart_title
-  - date_ap_style
\ No newline at end of file
+  - date_ap_style
+  - field_group_link
```
Comment (trackleft) [inline modules/custom/az_news/az_news.info.yml:28]:
```suggestion
  - field_group_link
  
```

**Review by trackleft — CHANGES_REQUESTED** [review]

**Review by danahertzberg — APPROVED** [review]
This looks lovely!

**Review by trackleft — APPROVED** [review]

File: composer.json [hunk 3]
```diff
@@ -24,39 +24,40 @@
...
         "drupal/externalauth": "1.3",
         "drupal/field_group": "3.1",
+        "drupal/field_group_link": "^3.0@RC",
+        "drupal/media_library_form_element": "2.0",
         "drupal/media_library_theme_reset": "1.0",
+        "drupal/metatag": "1.15",
         "drupal/migrate_plus": "5.1",
         "drupal/migrate_tools": "5.0",
-        "drupal/pathauto": "1.8",
-        "drupal/token": "1.7",
```
Comment (joeparsons) [inline composer.json:44]: The sorting changes in this file may cause conflicts with other open PRs so we might want to consider merging this PR last today.

**Comment by smayden** [comment]
Realized that stories were not going into URL /news so add pathauto settings

**Review by tadean — APPROVED** [review]

File: modules/custom/az_news/config/install/views.view.az_news.yml [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+          relationship: none
+          group_type: group
+          admin_label: ''
+          order: DESC
+          exposed: false
+          expose:
+            label: ''
+          granularity: second
+          plugin_id: datetime
+      title: 'AZ News'
```
Comment (camikazegreen) [inline modules/custom/az_news/config/install/views.view.az_news.yml:171]:
```suggestion
      title: 'News'
```

**Comment by danahertzberg** [comment]
Minor nitpick: I would recommend removing the "Strong" field group to simplify. `psuedo-link` (misspelled) is not needed and we can move `strong` to the surrounding "Heading" field group.
Screen Shot 2021-01-22 at 10 41 43 AM

**Review by camikazegreen — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**